src/actions/config.js

```diff
@@ -24,21 +20,41 @@ export function applyDefaults(config) {
   return defaultsDeep(config, defaults);
 }
 
+export function validateConfig(config) {
+  if (!has(config, 'backend' || !config['backend'])) {
```

Comment:
I think the first closing parens is misplaced here.

You can shorten this and a few of the following conditions using `lodash.get` in the form of:

`if (!get(config, 'backend'))`

-or-

`if (!get(config, ['backend', 'name']))`